python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -828,6 +829,37 @@ def f():
     wait_for_pid_to_exit(gcs_server_pid, 10000)
 
 
+@pytest.mark.skipif(
+    enable_external_redis(), reason="Only valid when started without external redis"
+)
+@pytest.mark.parametrize(
+    "ray_start_with_env_vars",
```

Comment:
you don't need a separate fixture for this, search `generate_system_config_map` in this file and follow that pattern.

Reply:
done